h/views/oidc.py

```diff
@@ -19,8 +25,16 @@
 from h.services.jwt import TokenValidationError
 
 
-@view_defaults(request_method="GET", route_name="orcid.oauth.authorize")
-class AuthorizeViews:
+class AccessDeniedError(Exception):
+    """The user denied us access to their ORCID record."""
```

Comment:
The docstring still mentions ORCID, but I presume the plan is to reuse these exceptions for other providers?

Reply:
Yes, added a commit to remove the ORCID-specific language.